research/metrics/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: PCMDI - Metrics
+---
+
+# PCMDI Metrics Package: Preliminary Results
+
+### Extra-tropical Variability Modes
+Metrics for Modes of Variability Metrics for CMIP5/6 models. The Modes of Variability Metrics is developed at Program for Climate Model Diagnosis and Intercomparison (PCMDI) of Lawrence Livermore Natioanal Laboratory (LLNL). The metrics were developed by Jiwoo Lee, Ken Sperber, Peter Gleckler, Celine Bonfils, and Karl Taylor. 
+
+> Lee, J., Sperber, K.R., Gleckler, P.J., Bonfils, C.J. and Taylor, K.E., 2019. Quantifying the agreement between observed and simulated extratropical modes of interannual variability. Climate dynamics, 52: 4057-4089, [doi: 10.1007/s00382-018-4355-4][lee2019].
+
+#### CMIP5 Historical: [Amplitude][CMIP5_variability_amplitude] | [RMSE][CMIP5_variability_rmse]
+#### CMIP6 Historical [Amplitude][CMIP6_variability_amplitude] | [RMSE][CMIP6_variability_rmse]
+---
+
+### Monsoon Precipitation: Fractional Accumulation
+Monsoon metrics for CMIP5 models. The Metric is developed at Program for Climate Model Diagnosis and Intercomparison (PCMDI) of Lawrence Livermore Natioanal Laboratory (LLNL). The metrics were developed by Ken Sperber.
+
+> Sperber, K.R. and Annamalai, H., 2014. The use of fractional accumulated precipitation for the evaluation of the annual cycle of monsoons. Climate dynamics, 43, 3219-3244, [doi:10.1007/s00382-014-2099-3][sperber2004]
+
+#### CMIP5 Historical: [Onset][CMIP5_monsoon_sperber_onset] | [Decay][CMIP5_monsoon_sperber_decay] | [Duration][CMIP5_monsoon_sperber_duration]
+
+
+
+[lee2019]: https://link.springer.com/article/10.1007/s00382-018-4355-4
+[sperber2004]: https://doi.org/10.1007/s00382-014-2099-3
+[CMIP5_variability_amplitude]: https://oceanonly.llnl.gov/lee1043/web/test_dir/ipp_test_variability_modes/cmip5_v20190512/clickable_portrait.html
```

Comment:
@lee1043 it is a really bad idea to provide a live web link to ocean - rather migrate these plots across to the pcmdiweb machine for production. Ocean has been stood up for prototyping only, and I personally avoid linking to any of the live URLs from a live webpage (pointing to the server)

Reply:
@durack1 agreed. It was for prototyping and accidentally merged to master, which I took it down later. @mauzey1 I wonder is there a dedicated directory place in the pcmdi web server.

Reply:
@lee1043 We can set up a directory on pmcdi.llnl.gov to serve your files.

Reply:
@mauzey1 that would be great, thank you. Can I do rsync from other machine for that? @gleckler1 may interested too

Reply:
@mauzey1 @lee1043  That would be great.  rsync from crunchy to pcmdiweb would be straightforward.   @mauzey1 Another question we have for you is that in order to fully view what we are prototyping on GH it seems that we should use absolute links to plots rather than relative.   That way we'd be able verify at each step (lee1043.github.io/research/metrics to pcmdi.github.io/research/metrics to pcmdi.llnl.gov/research/metrics) that all of our interactive features are working.   Does that seem reasonable?

Reply:
@lee1043 @gleckler1 I have made a directory for hosting the files at https://pcmdi.llnl.gov/pmp-preliminary-results/

Reply:
@mauzey1 Thank you.   What is the path on pcmdiweb?

Reply:
@gleckler1 
/var/www/pcmdi-static/pmp-preliminary-results/

If you have the files in your home directory, then I can transfer them to there.

Reply:
@mauzey1 @lee1043  Excellent, thank you.  Both PG and JL are about to take vacation so this won't happen for a few weeks, but when we return it will and we'll keep you posted.

Reply:
@mauzey1 thanks for setting it up.

Reply:
@mauzey1 could you please copy files and directories in `/work/lee1043/www/web/test_dir` of crunchy to the `/var/www/pcmdi-static/pmp-preliminary-results/` of pcmdi server? It looks I do not have writing permission. Thanks.

Reply:
@lee1043 The files have been transferred.

Reply:
@mauzey1 thanks.